admin/environment-management/workspace-providers.md

```diff
@@ -4,17 +4,35 @@ description: Learn how workspace providers can minimize latency for users.
 state: beta
 ---
 
-Workspace Providers are a forthcoming feature that enables a single Coder
-deployment to provision and manage workspaces across multiple Kubernetes
-clusters and namespaces, including ones located in other geographies, regions,
-or clouds.
+Workspace Providers are logical groups of resources developers can deploy
```

Comment:
Pardon me for, I'm just thinking out loud here... I suggest rephrasing as one of the following:

* Workspace Providers are logical groups of resources into which developers can deploy workspaces
* Workspace Providers are logical groups of resources that contain developers' workspaces.

I am also not sure if we should be using "environments" in place of "workspaces" here?

This avoids [ending the sentence with a preposition](https://www.grammarly.com/blog/youve-been-lied-to-heres-why-you-absolutely-can-end-a-sentence-with-a-preposition/) (I am also not an English professor and had to look this up)

I would defer to our Word Nerd @khorne3, of course :nerd_face:

Reply:
I'd probably go with

> Workspace Providers are logical groups of resources *to* which developers can deploy workspaces